src/components/sectioningConfirm/SectioningConfirm.tsx

```diff
@@ -121,18 +163,47 @@ export default function SectioningConfirm({
             {({ removePlanByBarcode }) => (
               <div className="mt-8 space-y-12">
                 {Object.keys(layoutPlansByLabwareType).length > 0 && (
-                  <div className="space-y-4">
-                    <Heading level={3}>Source Labware</Heading>
-                    <DataTable
-                      data={sourceLabware}
-                      columns={[
-                        columns.barcode(),
-                        columns.highestSectionForSlot("A1"),
-                      ]}
-                    />
-                  </div>
-                )}
+                  <>
```

Comment:
What does the addition of <> here signify?

Reply:
In React, expressions must have one parent element. So as to wrap that entire segment within a parent <> </> is used, or we can use <React.Fragment> </React.Fragment> as a wrapper.